config/model_configs/diagnostic_edmfx_trmm_box.yml

```diff
@@ -15,7 +15,7 @@ moist: equil
 cloud_model: "quadrature_sgs"
 call_cloud_diagnostics_per_stage: true
 apply_limiter: false
-precip_model: "1M"
+precip_model: "0M"
```

Comment:
later in this file we need to change to `toml: [toml/diagnostic_edmfx.toml]`. Could you also double check if there is any other toml need to be changed?